python/python/bystro/ancestry/train.py

```
@@ -361,35 +361,37 @@
             f"expected {EXPECTED_NUM_POPULATIONS}"
         )
         raise ValueError(msg)
-    get_pop_from_sample = ancestry_df["Population"].to_dict()
+    get_pop_from_sample = ancestry_df["Population elastic ID"].to_dict()
     labels = pd.DataFrame(
         [get_pop_from_sample[s] for s in samples],
         index=samples,
-        columns=["population"],
+        columns=["Population elastic ID"],
     )
+    labels.rename(columns={'Population elastic ID': 'population'}, inplace=True)
```

Ruff

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)